packages/kernel-5.10/config-bottlerocket

```diff
@@ -77,11 +30,6 @@ CONFIG_EFI=y
 CONFIG_EFI_STUB=y
 CONFIG_EFI_MIXED=y
 
-# EFI video
-CONFIG_FB=y
-CONFIG_FB_EFI=y
-CONFIG_FRAMEBUFFER_CONSOLE_DEFERRED_TAKEOVER=y
```

Comment:
I'd like to keep the EFI related variables in the common config, since they apply to platforms like EC2 and ESXi when using EFI boot.

Reply:
That makes sense. These were added when initially adding more hardware support, but probably were not just hardware enablement.